brewtils/rest/client.py

```diff
@@ -203,31 +204,52 @@ def _load_config(args, kwargs):
 
         return spec.load_config(*[kwargs, renamed, positional, brewtils.plugin.CONFIG])
 
+    def can_connect(self, **kwargs):
+        # type: (**Any) -> bool
+        """Determine if a connection to the Beer-garden server is possible
+
+        Args:
+            kwargs: Keyword arguments to pass to Requests session call
+
+        Returns:
+            A bool indicating if the connection attempt was successful. Will
+            return False only if a ConnectionError is raised during the attempt.
+            Any other exception will be re-raised.
+
+        Raises:
+            requests.exceptions.RequestException:
+                The connection attempt resulted in an exception that indicates
+                something other than a basic connection error. For example,
+                an error with certificate verification.
+        """
+        try:
+            self.session.get(self.config_url, **kwargs)
+        except requests.exceptions.ConnectionError as ex:
+            if type(ex) == requests.exceptions.ConnectionError:
+                return False
+            raise
+
+        return True
+
     @enable_auth
-    def get_version(self, **kwargs):
+    def get_version(self, **_):
```

Comment:
`**_` 
^ Not sure if I like this. Everywhere else we use `**kwargs`, why did you change that?

Reply:
Also, are you just trying to drop it on the floor because we don't use them? That way it doesn't break anyone's code if they are still calling it?

Reply:
Yep, that's correct. If you define arguments that the function doesn't use the linters / IDE will complain about it. Defining the argument as a literal underscore is the way to explicitly say "I know I'm not using this, but I need it there anyway."

And yeah, the reason I didn't just chop them out is so anybody that's currently using them doesn't break.

However, thinking about it more, it's probably better that it *does* break, so they don't think it still does anything. What do you think?

Reply:
What are your thoughts on publishing a logging error that they are using a deprecated feature. Don't stop them, but log it so they one day might see it and fix their code. Then in a later release we can remove it? We have to ensure the v2.0 plugins work at the get go, so let's not force any code changes on day one.

Reply:
Sounds good to me, I'll update that.

Reply:
Added a `_deprecate` call if keyword arguments are passed.